plugins/inputs/vsphere/vsphere.go

```diff
@@ -179,21 +181,24 @@ func init() {
 			DatastoreMetricInclude:    nil,
 			DatastoreMetricExclude:    nil,
 			DatastoreInclude:          []string{"/*/datastore/**"},
+			VSANMetricInclude:         nil,
+			VSANMetricExclude:         nil,
```

Comment:
Shouldn't it be like this according to the README saying this is the default?
```suggestion
			VSANMetricExclude:         []string{"*"},
```